### ISSUE TITLE
language chooser is not working

### ISSUE THREAD

**Issue by verdulo** [AndBible/and-bible#481]
**Describe the bug**
Device language is Polish, language of program is set to Esperanto, most of texts are in Polish.

**Bug was found on And Bible version**
3.1.319.4-beta (F-Droid)
Problem not occurred in previous version (2.11.1.246).

**Expected behavior**
See the program in Lingvo Internacia.

**Screenshots**
![bildo](https://user-images.githubusercontent.com/16842967/71491798-29537580-2833-11ea-861c-670cc9e568b0.png)

**Smartphone (please complete the following information):**
 - Device: Lenovo a2016
 - OS: Android
 - Version 6.0

**Comment by tuomas2** [comment]
Hi @verdulo 
Polish translation is way out of date and needs to be updated. 
Could you help with translation? It can be done in Transifex: 

https://www.transifex.com/mjdenham/andbible/language/pl/

If we don't get updated translation, it should be considered if it is better to remove the translation
alltogether, rather than keep it in half-translated state.

**Comment by verdulo** [comment]
I can help translating to Polish. But this bug report is not about quality of Polish translation, I want to see the program in Esperanto using the in-program option to change the language of interface, in the previous version (2.11.1.246) it worked, but now it isn't.

**Comment by tuomas2** [comment]
Hmm interesting. When I change to Esperanto, I see something like this: (screenshot comes next)

**Comment by tuomas2** [comment]
Tried with emulator (Android 10), could not reproduce. Changed system language to polish and then And Bible language to Esperanto and got the following:
![Screenshot_1577437522](https://user-images.githubusercontent.com/5811789/71510829-ddd3b280-2898-11ea-9a86-0c24c465b3a0.png)

**Comment by verdulo** [comment]
EO
Mi kontrolis la lingv‑elektilon ĉe kelkaj telefonoj: **ĉe telefonoj kun Android 4.4 ĉio funkcias bone**, sed ĉe Lenovo a2016 kun Android 6.0 kelkaj elementoj restis en lingvo de la sistemo.
Mi uzas la Xposed-modulon [App locale 2](https://github.com/Flo354/XposedAppLocale) kaj tiu problemo rilatas ankaŭ al aliaj aplikaĵoj. Tamen mi trovis provizoran solvon: ŝanĝi lingvon (per la menciita ilo) de aplikaĵo “Android system” al eo kaj ŝanĝi lingvon de aliaj aplikaĵoj (“System UI”, “Settings”, “Clock”, “Gboard”, ktp) kiuj ne havas EO-tradukon al pl.

EN
I checked the language picker on a few phones: **on phones running Android 4.4 everything works fine**, but on Lenovo a2016 with Android 6.0 a few elements remained in the system language.
I am using the Xposed module [App locale 2] (https://github.com/Flo354/XposedAppLocale) and this problem is related to other applications as well. However, I found a temporary solution: change the language (using the tool mentioned above) of the "Android system" application to en and change the language of other applications ("System UI", "Settings", "Clock", "Gboard", etc.) which not have EO translation to pl.
